pgdrive/utils/coordinates_shift.py

```diff
@@ -30,7 +30,9 @@ def panda_position(position, z=0.0) -> Vec3:
     """
     if len(position) == 3:
         z = position[2]
-    return Vec3(position[0], -position[1], z)
```

Comment:
why do this? PandaPosition method is expected to return a tuple in panda data structure